Dockerfile

```diff
@@ -240,3 +240,24 @@ ARG deny_version=0.7.0
 ARG deny_location=https://github.com/EmbarkStudios/cargo-deny/releases/download/${deny_version}/cargo-deny-${deny_version}-x86_64-unknown-linux-musl.tar.gz
 RUN curl --location ${deny_location} | tar --extract --gzip --directory=${install_dir} --strip-components=1
 RUN chmod +x ${install_dir}/cargo-deny
+
+# Unset $CARGO_HOME so that the new user will use the default value for it, which will point it to
+# its own home folder.
+ENV CARGO_HOME ""
+
+# Placeholder args that are expected to be passed in at image build time.
+# See https://code.visualstudio.com/docs/remote/containers-advanced#_creating-a-nonroot-user
+ARG USERNAME=user-name-goes-here
```

Comment:
What happens if no value is assigned here? If that produces a meaningful error message when building the image, it might be better not to assign a value here.

Reply:
In that case I agree that it is better to assign a value like this :)